app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestViewModel.kt

```diff
@@ -96,60 +94,46 @@ class MeetingRequestViewModel(
    *
    * @param newToken the current token of the user to which we want to send the message
    */
-  fun onLocalTokenChange(newToken: String) {
+  fun setTargetToken(newToken: String) {
     meetingRequestState = meetingRequestState.copy(targetToken = newToken)
   }
   /**
    * Sets the message of the meeting request
    *
    * @param newMessage the message we want to send
    */
-  fun onMeetingRequestChange(newMessage: String) {
-    meetingRequestState = meetingRequestState.copy(message = newMessage)
+  fun setMeetingRequestChangeFirstMessage(message1: String) {
+    meetingRequestState = meetingRequestState.copy(message1 = message1)
   }
+
   /**
-   * Sets the message of the meeting response
+   * Sets the second message of the meeting request
    *
-   * @param targetToken: the FCM token of the target user
-   * @param newMessage: the response message we want to send
-   * @param accepted: the acceptation status of the meeting request
+   * @param message2 the message we want to send
    */
-  fun setMeetingResponse(targetToken: String, newMessage: String, accepted: Boolean) {
-    meetingResponseState =
-        meetingResponseState.copy(
-            targetToken = targetToken, message = newMessage, accepted = accepted)
+  fun setMeetingRequestChangeSecondMessage(message2: String, location: String) {
+    meetingRequestState = meetingRequestState.copy(message2 = message2, location = location)
   }
 
   /**
-   * Sets the message of the meeting cancellation
+   * Sets the message of the meeting response
    *
    * @param targetToken: the FCM token of the target user
-   * @param cancellationReason: the reason for the cancellation of the meeting request
-   * @param otherUserName: the name of the other user
+   * @param newMessage: the response message we want to send
+   * @param accepted: the acceptation status of the meeting request
```

Comment:
You forgot `location` in the documentation

Reply:
Thank you ! good catch